src/utility.js

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from "react";
+
+const API_Company = "https://match-labs-api.herokuapp.com/api/no_auth/labs";
```

Comment:
This represents the "Lab" API, not the "Company" API.

As a rule of thumb, API endpoints should be pluralised since they return a bunch of resources. Also, the components don't have to know about the API host. You could have that stored internally, and only expose `/labs` and `/candidates` as constants. This has the added advantage that if the API host changes, you'll only have to edit it in one place!